src/api/v1/server.go

```diff
@@ -10,6 +10,7 @@ func API(api *echo.Group) {
 	// TODO API Doc
 
 	api.GET("/motd", getMotd, middleware.Cache(60*30))
+	api.GET("/themes", getThemes, middleware.CacheUntilRestart(60*60*24 /*1day*/))
```

Comment:
This really only makes sense for browsers since Java doesn't have a local http cache. This will have an infinite s-maxage for cloudflare (good) but a 1 day max-age (ignored). Idk the best way to express this tho... Maybe a CacheUntilRestart() which is public, s-maxage=1year, max-age=0? With the intent that it's only to be used for the client not the website? Idk